impeller/entity/entity_pass.cc

```diff
@@ -41,7 +41,7 @@ void EntityPass::AddEntity(Entity entity) {
   if (entity.GetBlendMode() > Entity::BlendMode::kLastPipelineBlendMode) {
     reads_from_pass_texture_ = true;
   }
-
+  cover_whole_screen |= entity.CoverWholeScreen();
```

Comment:
Per my other comment, I don't think propagating this expansion to the parent pass is necessary to achieve the correct behavior. This line in both `AddEntity` and `AddSubpass` can probably be removed.

Reply:
Done